tests/test_lightningd.py

```diff
@@ -4205,7 +4226,9 @@ def test_peerinfo(self):
         assert l1.rpc.getpeer(l2.info['id'])['color'] == l1.rpc.listnodes(l2.info['id'])['nodes'][0]['color']
 
         # Close the channel to forget the peer
-        l1.rpc.close(l2.info['id'])
+        self.assertRaisesRegex(ValueError,
```

Comment:
I don't understand why now all of these closes fail in the integration tests. Aren't these supposed to be collaborative closes? The 0 timeout everywhere is a bit excessive IMHO

Reply:
The tests assume that `close` does not wait for the close to complete.  This PR makes `close` wait either for the timeout or the close to complete; to emulate the old behavior we set timeout to 0 and let the command fail.  Otherwise I would have to check each instance of `.rpc.close` to see if I can remove the `wait_for_logs` afterwards or if they scan for something more substantial than bilateral close.

Reply:
Well, ok, we should slowly make these closes normal again though (noting this for my cleanup queue).